pkgs/build-support/build-bazel-package/default.nix

```
@@ -50,13 +53,16 @@ let
     bazelFlags = bazelFlags;
     bazelBuildFlags = bazelBuildFlags;
     bazelTestFlags = bazelTestFlags;
+    bazelRunFlags = bazelRunFlags;
+    runTargetFlags = runTargetFlags;
     bazelFetchFlags = bazelFetchFlags;
     bazelTestTargets = bazelTestTargets;
+    bazelRunTarget = bazelRunTarget;
     dontAddBazelOpts = dontAddBazelOpts;
```

```suggestion
    inherit bazelFlags bazelBuildFlags bazelTestFlags bazelRunFlags runTargetFlags bazelFetchFlags bazelTestTargets bazelRunTarget dontAddBazelOpts;
```
please also include name

---

Done !